default/stringtables/en.txt

```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
 **Quick Start Guide**
 
 QUICK_START_GUIDE_TEXT
-'''This is a brief introduction to FreeOrion gameplay. A somewhat less brief Quick Play Guide is available at freeorion.org (current URL: <url  http://freeorion.org/index.php/V0.4_Quick_Play_Guide>freeorion.org/index.php/V0.4_Quick_Play_Guide</url> ).
+'''The Pedia is ment to assist the player with and provide reference material about all aspects of this game, its mechanics and its various content. Right-clicking on many items on the game screen will lead to specific Pedia articles. And now, to get started.
 
-[[DETAILED_PEDIA_REFERENCE]]
-Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]], [[encyclopedia CATEGORY_GUIDES]], and the [[encyclopedia ENC_METER_TYPE]] list.
+There are three urgent problems that need to be addressed.
+
+1. All starships are initially idle in orbit around the central star of the home world's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system highlighted in the center. Next to it, a bit up and to the right, is the fleet. Ships can be selected by left-clicking on the fleet symbol and then given march orders by right-clicking a target star. If ships get no orders, they will stay idle. Scouts should be sent out to uncover more of the map. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```

Comment:
```suggestion
1. Fleets are initially idle at homeworld's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system selected and highlighted in the center. Next to it, a bit up and to the right, is the fleets icon. Fleets can be selected by left-clicking on the fleet symbol on the map, which will open the fleets window, where all the fleets in a system and their contained ships can be viewed. If a fleet is selected, it can be given movement orders by right-clicking a nearby target star that is connected to the fleet's system by a starlane. Fleets that are not given move orders will stay idle. A good first step is often to send scout fleets to explore nearby systems, which will reveal more about their contents and any further starlane connections beyond them. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```